ui/raidboss/data/03-hw/alliance/weeping_city.js

```diff
@@ -0,0 +1,312 @@
+      netRegex: NetRegexes.message({ line: '.*Queen\'s Room will be sealed off.*?', capture: false }),
+      run: function(data) {
+        data.arachneStarted = true;
+      },
+    },
+    {
+      id: 'Weeping City HeadMarker Ozma',
+      netRegex: NetRegexes.message({ line: '.*Gloriole will be sealed off.*?', capture: false }),
+      run: function(data) {
+        data.arachneStarted = false;
+        data.ozmaStarted = true;
+      },
+    },
+    {
+      id: 'Weeping City HeadMarker Calofisteri',
+      netRegex: NetRegexes.message({ line: '.*Tomb Of The Nullstone will be sealed off.*?', capture: false }),
+      run: function(data) {
+        data.ozmaStarted = false;
+        data.calStarted = true;
+      },
+    },
+    {
+      id: 'Weeping City Sticky Wicket',
+      netRegex: NetRegexes.headMarker({ id: '003C', capture: false }),
+      suppressSeconds: 10,
+      response: Responses.spread(),
+    },
+    {
+      id: 'Weeping City Shadow Burst',
+      netRegex: NetRegexes.headMarker({ id: '003E' }),
+      condition: function(data) {
+        return data.arachneStarted;
+      },
+      response: Responses.stackOn(),
+    },
+    {
+      id: 'Weeping City Frond Affeared',
+      netRegex: NetRegexes.startsUsing({ id: '183A', source: 'Arachne Eve', capture: false }),
+      response: Responses.lookAway(),
+    },
+    {
+      id: 'Weeping City Arachne Web',
+      netRegex: NetRegexes.headMarker({ id: '0017' }),
+      condition: function(data, matches) {
+        return data.arachneStarted && data.me == matches.target;
+      },
+      infoText: {
+        en: 'Drop Web Outside',
+      },
+    },
+    {
+      id: 'Weeping City Brand Of The Fallen',
+      netRegex: NetRegexes.headMarker({ id: '0037' }),
+      condition: Conditions.targetIsYou(),
+      response: Responses.doritoStack(),
+    },
+    {
+      id: 'Weeping City Dark Eruption',
+      netRegex: NetRegexes.headMarker({ id: '0019' }),
+      condition: Conditions.targetIsYou(),
+      infoText: {
+        en: 'Puddles on YOU',
+        de: 'Pfützen auf DIR',
+        fr: 'Mare sur VOUS',
+        ja: '自分に床範囲',
+        cn: '点名',
+        ko: '장판 바깥에 깔기',
+      },
+    },
+    {
+      id: 'Weeping City Beguiling Mist',
+      netRegex: NetRegexes.startsUsing({ id: '17CE', source: 'Summoned Succubus' }),
+      condition: function(data) {
+        return data.canSilence();
+      },
+      response: Responses.interrupt(),
+    },
+    {
+      id: 'Weeping City Mortal Ray',
+      netRegex: NetRegexes.startsUsing({ id: '17D4', source: 'Summoned Haagenti' }),
+      response: Responses.lookAwayFrom(),
+    },
+    {
+      id: 'Weeping City Hell Wind',
+      netRegex: NetRegexes.startsUsing({ id: '17CB', source: 'Forgall', capture: false }),
+      // Hell Wind sets HP to single digits, so mitigations don't work. Don't notify non-healers.
+      condition: function(data) {
+        return data.role == 'healer';
+      },
+      response: Responses.aoe(),
```

Comment:
What text is used on Kefka for this? I wonder if we need something different for these sorts of mechanics than "aoe"

Reply:
It looks like neither Exdeath nor Kefka have triggers for this. The only similar mechanic I can think of right now is Charybdis on Alte Roite, and that uses this same trigger construction for both Normal and Savage. I'm fully on board with "HP-to-1 attack incoming!" or something similar.

Reply:
Maybe we should add something in Responses for this (and those?) I don't know what to call it, though.  People didn't like e2s being called protean, so maybe we shouldn't call this charybdis or heartless archangel or whatever.

Reply:
Perhaps "Unblockable raid damage" or something like that. I'll put something together after we land this one?

Reply:
Yeah, a followup seems reasonable!